cmd/router/main.go

```diff
@@ -109,6 +113,37 @@ func routeStep(nodeName string, graph v1alpha1.InferenceGraphSpec, input []byte,
 		}
 		return executeStep(route, graph, input, headers)
 	}
+	if currentNode.RouterType == v1alpha1.Mirroring {
+		//* check the step len, if step is nil, return input directly
+		if currentNode.Steps == nil {
+			return input, nil
+		}
```

Comment:
I think there should be some validation on the inference graph yaml that the steps should be more than two for mirroring node type

Reply:
add a validation for mirroring spec